src/main/java/com/github/manolo8/darkbot/core/manager/EffectManager.java

```diff
@@ -60,15 +60,132 @@ public enum Effect {
         UNDEFINED(-1),
         LOCATOR(1),
         PET_SPAWN(2),
+        SOLACE_NANO_CLUSTER(3),
+        DIMINISHER_WEAKEN_SHIELDS(4),
+        SPECTRUM_PRISMATIC_SHIELD(5),
+        SENTINEL_FORTRESS(6),
+        /** Cyborg uses the same effect as venom for singularity */
+        VENOM_SINGULARITY(7),
+        SABOTEUR(9),
+        /* SHIELD_1(10), */
         ENERGY_LEECH(11),
+        REPAIR_BATTLE_BOT(12),
+        /* LEVEL_UP(13), */
+        LIGHTNING_ABILITY(14),
+        HEAL_RAY(15), // AEGIS_HP_REPAIR RAY
         NPC_ISH(16),
-        DRAW_FIRE(36),
+        /* RAGE(17),
+        SKULL(18),
+        SPAWN(19), */
+        BOX_COLLECTING(20),
+        BOOTY_COLLECTING(21),
+        /* STATIC_MAP(22),
+        ACHIEVEMENT(23),
+        SPAWN_CHILDREN(24),
+        IS_SPAWNED(25),
+        UBER(26),
+        ABSORBATION(27),
+        ABSORBATION_ELITE(28), */
+        LEONOV_HOME_BONUS(29),
+        SHIELD_ENGINEERING_SKILL(30),
+        CHARGE_SHIELD_RAY(31),
+        /* MARKED_BY_BEACON(32),
+        STANDALONE_RING_POD(33),
+        DAMAGE_TRANSFER_RAY(34), */
+        CITADEL_FORTIFY(35),
+        CITADEL_DRAW_FIRE(36),
+        CITADEL_PROTECTION(37),
+        PROTECTED(38),
+        CITADEL_DRAW_FIRE_VICTIM(39),
+        /* MOUSE_CURSOR_CHANGE(40), */
+        WARP_ANIMATION(41), //Animation when changing ship
+        /* INVASION_NPC(42),
+        SOLAR_GLOW(43),
+        SATURN_EVIL_GLOW(44),
+        DEFLECTOR_SHIELD(45),
+        EMERGENCY_REPAIR(46),
+        CONSTRUCTION(47),
+        SELF_HEAL(48), */
+        COOLDOWN_TIMER(49),
```

Comment:
```suggestion
        /** Used by several things to display a timer on the ship, like x, y and z */
        COOLDOWN_TIMER(49),
```
This one needs to be explained or at least use an example on a comment